UnitsConverter.Fluent.Plugin/ConvertUtil.cs

```diff
@@ -27,6 +27,82 @@ internal class ConvertUtil
             QuantityType.Volume,
         };
 
+        public static List<IQuantity> ConvertWithoutQuantity(string text)
+        {
+            // The text is in the form of "5 lb in kg" or "5 lbs to kg"
+            string[] parts = text.Split(new string[] { "in", "to" }, StringSplitOptions.RemoveEmptyEntries);
+            string from = parts[0].Trim();
+            string to = parts.Length > 1 ? parts[1].Trim() : null;
+            
+            if (Acceleration.TryParse(from, out Acceleration accel))
```

Comment:
If `QuantityType` names are matching all the other enums you can do something like this -
`new List<Enum> { typeof(Area), typeof(Duration), ...}.FirstOrDefault(t => Enum.TryParse(t, a, out var _) && QuantityType.TryParse(t.Name, out quantityType)?.Name`
If you do keep that list as a static field.

Reply:
Adir, I have removed this class and add all logic in UnitHandler. I may need to add some of the logic back to support this term "5 meters in" without a given "toUnit".